homepage.css

```diff
@@ -20,61 +24,77 @@ body{
 
 nav{
 	width:100%;
-	background:white;
+	background-color:#0C59CF;
 	overflow:auto;
+	
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
 	list-style: none;
 }
 
 li{
-	float:left;
+	float:right;
 }
 
 
+
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
 	transition:0.5s;
 }
 
 ul li ul{
 	padding: 0;
 	position: absolute;
 	top: 58px;
-	left: 130px;
 	width: 100px;
 	display: none;
 	visibility: hidden;
-	z-index: 1;
+	z-index: 5;
+}
+
+ul li #a{
```

Comment:
26/8/2021: This is for the drop down menu for the setting which in the admin nav but for user they can't access to the setting because user don't have setting option.